.example.env

```diff
@@ -1,4 +1,14 @@
+# {Number} - Server port
 PORT=
+
+# {String} - Server host
 HOST=
+
+# {String} - a Postgress database URL
 DATABASE_URL=
+
+# {Boolean} - Allow introspection GraphQL server
 INTROSPECTION_ENABLED=
+
+# {String} - a Postgress database URL to use as a shadow database for prisma migration
+SHADOW_DATABASE_URL=
```

Comment:
you need to add this to src/configuration/validation.ts also and src/configurations/EnvironmentVariables.ts

Reply:
We do not use this one in runtime environment.